documentation/development/plans/2026-04-22-visual-facelift-implementation.md

```
@@ -0,0 +1,1890 @@
+done
+```
+
+- [ ] **Step 2: Gör punktersättningar för kändra RODA-RODA-ord**
+
+Öppna filen i editor. För varje träff nedan, ersätt värde (höger om `=`) enligt tabell:
+
+| Hitta (svensk text idag) | Byt till |
+|--------------------------|----------|
+| `Instrumentpanel` eller `Dashboard` | `Översikt` |
+| `Intag` eller `Ingest` i rubrik | `Leverans` |
+| `Lämna in` — behåll |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)